drivers/hwinfo/Kconfig

```diff
@@ -0,0 +1,41 @@
+# Kconfig -HW Info driver configuration options
+
+#
+# Copyright (c) 2019 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config HWINFO
+	bool
+
+menuconfig DEVICE_ID
```

Comment:
The `DEVICE_ID` option is inconsistent with the driver name. It should be renamed to `HWINFO`. Remaining Kconfig options in this file should be updated accordingly.

Reply:
Done